scopes.md

```diff
@@ -0,0 +1,4 @@
+# Scopes expected by pushsnapscript
+
+* `project:releng:snapcraft:firefox:{candidate,beta,mock}`: This represents the Snap store channel pushsnap script publishes the snap to. pushsnapscript only accepts a subset of what the [Snap store allows](https://docs.snapcraft.io/reference/channels#risk-levels-meaning).
```

Comment:
are you able to either link to or move the `(nightly, beta, release) vs (edge, beta, candidate, release)?` section here?

Reply:
Good idea. Moved.